sandbox/grist/functions/math.py

```diff
@@ -859,7 +860,13 @@ def TRUNC(value, places=0):
   return ROUNDDOWN(value, places)
 
 def UUID():
-  """Generate a random UUID-formatted string identifier."""
+  """
+  Generate a random UUID-formatted string identifier. 
+  Since UUID() produces a different value each time it's called, it is best to use it in 
+  [trigger formula](https://support.getgrist.com/formulas/#trigger-formulas) for new records. 
```

Comment:
When making links within documentation, there is a special pattern: use the name of the documentation file within parentheses, e.g. `formulas.md` here, so the full link will look like `[trigger formula](formulas.md#trigger-formulas)`. This will get translated to a correct link within the same site (i.e. support.getgrist.com vs support-preview.getgrist.com vs localhost when testing locally). The full URL will take you always to the production site, which makes it harder to verify changes.